cluster/src/test/resources/log4j.properties

```diff
@@ -74,3 +74,5 @@ log4j.logger.org.apache.spark.scheduler.FairSchedulableBuilder=WARN
 log4j.logger.org.apache.spark.scheduler.cluster.CoarseGrainedSchedulerBackend$DriverEndpoint=WARN
 log4j.logger.org.apache.spark.storage.BlockManagerInfo=WARN
 log4j.logger.org.apache.spark.executor.SnappyExecutor=WARN
+
+# log4j.logger.org.apache.spark.sql.execution.WholeStageCodegenExec=DEBUG
```

Comment:
Yes, because this is required frequently while debugging so added a commented one for ease.